app/views/static_pages/about.html.erb

```diff
@@ -0,0 +1,18 @@
+<% provide(:title, "About") %>
+<h1>About</h1>
+<p>
+  <%= link_to t("english") , locale: "en" %>
+  <%= link_to t("vietnam") , locale: "vi" %>
+  The <a href="https://www.railstutorial.org/"><em>Ruby on Rails
+  Tutorial</em></a>, part of the
+  <a href="https://www.learnenough.com/">Learn Enough</a> family of
```

Comment:
Dùng tag `link_to` thay vì `a` nhé

nhớ I18n cho toàn bộ TEXT của tất cả các file html nhé